src/test/suite/explorer/helpExplorer.test.ts

```diff
@@ -55,12 +55,6 @@ suite('Help Explorer Test Suite', function () {
     const testHelpExplorer =
       mdbTestExtension.testExtensionController._helpExplorer;
 
-    sinon.replace(
-      mdbTestExtension.testExtensionController._telemetryController,
-      'trackLinkClicked',
-      sinon.fake.resolves()
-    );
-
```

Comment:
Since now all tests have telemetry disabled we don't need anymore to fake telemetry events individually.